craft_application/commands/other.py

```diff
@@ -18,7 +18,7 @@
 
 from craft_cli import CommandGroup, emit
 
-from craft_application.commands import base
+from . import InitCommand, base
```

Comment:
In that case why not put `InitCommand` into this file?

Reply:
I'm personally ok with leaving `InitCommand` in its own file, to keep this `other.py`'s size under control

Reply:
That, keeping the test files under control, and I also like how the command and its tests are detached and easy to find in the file tree.